### DISCUSSION TITLE
s/ljawb7yb5o11qczx9aeqrq

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#8] [Announcements]
# s/ljawb7yb5o11qczx9aeqrq

游记 | 你可能从未看过这样的千岛湖

https://blog.ops-coffee.cn/s/ljawb7yb5o11qczx9aeqrq

**Comment by ghost** [comment]
土生土长的千岛湖人过来评论一下  
的确是这样的，环湖是游千岛湖的最佳选择，特别是骑行，每个季节和天气都会有不一样的风味，比上岛去玩有意思多了。  
还有民宿是千岛湖的一大特色，环湖一圈又很多优秀的小众精品民宿，本人就在一个离镇上40km远的偏僻民宿里打过工(很愉快的经历)，不得不说开这种民宿的都是有故事。  
吃的话千岛湖本地餐馆都是偏辣口的，不会像很多大排档网红餐馆一样，特别是浓汤鱼头，千岛湖人是不吃的。

**Reply by ops-coffee** [reply]
哈哈，没想到还有当地人来给加buffer，感谢你的分享，很有用
